test/modules/encoders/test_clip_text_encoder.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from test.test_utils import assert_expected, set_rng_seed
+from torchmultimodal.modules.encoders.clip_text_encoder import CLIPTextEncoder
+
+
+class TestCLIPTextEncoder:
+    @pytest.fixture
+    def setUp(self):
+        set_rng_seed(1234)
+        context_length = 77
+        return context_length
+
+    def test_clip_init(self, setUp):
+        context_length = setUp
+        batch_size, embedding_dim = 4, 8
+        text_length = min(context_length, 3)
+        text = torch.randint(1, 10, (batch_size, text_length), dtype=torch.long)
+
+        text_encoder = CLIPTextEncoder(embedding_dim=embedding_dim, use_clip_init=True)
```

Comment:
A nice thing to do here to confirm that `use_clip_init` is actually doing something (as opposed to e.g. parameters being initialized as all zeros) would be to set `use_clip_init=False` and assert that the mean is nonzero, then call `initialize_parameters` directly and run the zero-mean assertions